tests/integration/courses.test.js

```diff
@@ -1115,6 +1115,28 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
       expect(response.statusCode).toBe(404);
     });
 
+    it('should set expectedBillsCount to a lower value if not lower than number of validated bills', async () => {
```

Comment:
Que penses tu de retiirer "to lower value"  et laisser "should set expectedBillsCount if not lower than number of validated bills" ?